src/foam/dashboard/view/DashboardCitationView.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.dashboard.view',
+  name: 'DashboardCitationView',
+  extends: 'foam.u2.View',
+
+  axioms: [
+    foam.pattern.Faceted.create()
+  ],
+
+  imports: [
+  'stack'
+  ],
+
+  exports: [
+    'as rowView'
+  ],
+
+  properties: [
+    {
+      class: 'Class',
+      name: 'of'
+    }
+  ],
+
+  methods: [
+    function initE() {
+      var self = this;
+      this
+        .on('click', function() {
+          self.openFilteredListView(self.data);
+        })
+        .addClass(this.myClass())
+        .start()
+          .addClass('id')
```

Comment:
Does this work? Does the CSS get applied? Do you need to do this.myClass('id')? Same below.

Reply:
not applied either way..